tests/test_driver.py

```diff
@@ -146,9 +146,9 @@ def test_get_empty(self, driver, query_params):
         response = driver.transactions.get(asset_id='a' * 64)
         assert response == []
 
-    @mark.parametrize('operation,tx_qty', (
+    @mark.parametrize('operation,tx_qty', [
         (None, 3), ('CREATE', 1), ('TRANSFER', 2)
-    ))
+    ])
```

Comment:
During debugging we tried to have a single nested tuple i.e. `((None,3))` and python complained about it. So, we moved to a list of tuples.

Reply:
Because `((None, 3)) == (None, 3) != ((None, 3), )`. In general having lists can be less error prone. If you have a tuple with just one element you need to remember the trailing comma, that is :no_good_man: